storage/libradb/src/ledger_store/mod.rs

```diff
@@ -61,12 +61,11 @@ impl LedgerStore {
         }
     }
 
-    /// Return the ledger infos with their least 2f+1 signatures starting from `start_version` to
+    /// Return the ledger infos with their least 2f+1 signatures starting from `start_epoch` to
     /// the most recent one.
     /// Note: ledger infos and signatures are only available at the last version of each earlier
     /// epoch and at the latest version of current epoch.
-    #[cfg(test)]
-    fn get_ledger_infos(
+    pub fn get_latest_ledger_infos_per_epoch(
```

Comment:
the rename seems unnecessary? (especially when you still call it the old way in `storage_client` etc.

Reply:
The name get_ledger_infos() wasn't very intuitive to me. What about get_latest_ledger_infos()? We could go to get_latest_ledger_infos_per_epoch(), that would make it even more intuitive, but I don't have strong opinions there...

Reply:
`get_last_ledger_infos()`?